Websites/webkit.org/wp-content/themes/webkit/standards-positions.php

```diff
@@ -0,0 +1,176 @@
+
+@media only screen and (max-width: 1200px) {
+  #content > table th:nth-child(2),
+  #content > table td:nth-child(2) { width:10em }
+}
+@media only screen and (max-width: 1024px) {
+  #content > table th:nth-child(5),
+  #content > table td:nth-child(5) { display:none }
+  #content > table th:nth-child(6),
+  #content > table td:nth-child(6) { display:none }
+}
+
+@media only screen and (max-width: 800px) {
+  #content > table th:nth-child(3),
+  #content > table td:nth-child(3) { display:none }
+}
+</style>
+
+<main id=content>
+  <h1><a href="/">Standards Positions</a></h1>
+
+  <p>Enable JavaScript for an interactive summary table of WebKit's standards positions. Failing that, browse the <a href="https://github.com/WebKit/standards-positions">standards-positions GitHub repository</a> directly.</p>
+</main>
+
+<script>
+(function () { // Begin closure
+
+function createElement(name, attributes = {}, child=null) {
+  const element = document.createElement(name);
+  for (const name in attributes) {
+    element.setAttribute(name, attributes[name]);
+  }
+  if (child !== null) {
+    element.append(child);
+  }
+  return element;
+}
+
+function addElement(parent, name, attributes = {}, child=null) {
+  return parent.appendChild(createElement(name, attributes, child));
+}
+
+function createStandardsPositionsTable() {
+  const fragment = new DocumentFragment();
+  const p = addElement(fragment, "p", { id: "filter-container" });
+  const input = addElement(p, "input", { class: "search-input", type:"search", id: "filter", placeholder: "Filter specifications…" });
+  input.oninput = updateStandardsPositionsTableRows;
+  const topicLabel = addElement(p, "label", {}, "By topic:");
+  const topicSelect = addElement(topicLabel, "select", { id: "filter-topic" });
+  topicSelect.oninput = updateStandardsPositionsTableRows;
+  const table = addElement(fragment, "table");
+  const headerRow = addElement(addElement(table, "thead"), "tr");
+  for (const heading of ["Issue", "Specification", "Concerns", "Position", "Topics", "Venues", "Self-link"]) {
+    addElement(headerRow, "th", { scope: "col" }, heading);
+  }
+  addElement(table, "tbody");
+  return fragment;
+}
+
+function createFilterTextRE(filterText) {
+  if (filterText) {
+    try {
+      return new RegExp(filterText, "i");
+    } catch(e) {}
+  }
+  return null; 
+}
+
+function createStandardsPositionsTableRows(issues, filterText = "", filterTopic = "") {
+  const fragment = new DocumentFragment();
+  const filterTextRE = createFilterTextRE(filterText);
+  for (const issue of issues) {
+    if (filterTextRE && issue["title"].search(filterTextRE) === -1) {
+      continue;
+    }
+    if (filterTopic && !issue["topics"].includes(filterTopic)) {
+      continue;
+    }
+    const id = issue["id"].substring("https://github.com/WebKit/standards-positions/issues/".length);
+    
+    const row = addElement(fragment, "tr", { id: `position-${id}` });
```

Comment:
This really feels like it should be coming from a HTML `<template>` element.

Reply:
This is the style we use elsewhere on webkit.org. I'd rather not deviate much from it. And also, `<template>` should have variable support first.